docs/az-aro-python-development.md

```diff
@@ -9,11 +9,20 @@ There are currently two codebases for the `az aro` command:
   [upstream](https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules/aro)
   `az aro` module.
 
-The upstream `az aro` module is distributed with `az` and is automatically
-present in the Azure cloud shell.  The downstream extension can be installed by an
-end user to override the module (e.g. to fix an issue rapidly).  We use the
-extension for development and testing of new API versions.  Customers are
-advised to use the upstream CLI.
+The upstream `az aro` command module is distributed with `az` and is
+automatically present in the Azure cloud shell. Development/maintenance of this
+module within the Azure CLI is handled the same as any other first-party Azure
+CLI module. You can read more about how command modules are authored and
+maintained [here](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules).
+
+The downstream extension can be installed by an end user to override the module
+(e.g. to fix an issue rapidly).  We also use the extension directly from this
```

Comment:
nit: I think `to use preview features` is a more common usage for the overrides, but this works too and is clear.

Reply:
changed